projects/dev-schematics/src/shared/constants.ts

```diff
@@ -0,0 +1,13 @@
+/***** Imports start *****/
```

Comment:
Related to this comment: https://github.com/SAP/spartacus/pull/7554/files#r426770948: we can also expose the whole `constants.ts` file from `schematics` project, in which case we don't need this one. Therefore, we can move `TEST_OUTLET_MODULE` and `TEST_CONFIG_MODULE` constants there.